File: devices/controller/sht1x/read_dev.js

exports.read = deviceOptions => {
    return new Promise((resolve, reject) => {
        resolve({httpCode: 200, type: 'SUCCESS', data: [{"type": "TEMPERATURE", "value": "22"}, {"type": "HUMIDITY", "value": "45"}, {"type": "DEW", "value": "8"}], message: 'Read sht10 successfuly from dataPin: '+deviceOptions.dataPin+', clockPin: '+deviceOptions.clockPin})
    })
}

File: devices/controller/sht1x/read_dev.py
import sys

print '{"httpCode": "200", "type": "SUCCESS", "message": "All good from data pin '+sys.argv[1]+' and clock pin '+sys.argv[2]+'.", "data": [{"type": "TEMPERATURE", "value": "20.42"}, {"type": "HUMIDITY", "value": "42.20"}]}'
